website/database_structure/_03_Samples.md

```
@@ -50,7 +50,7 @@ information</span> is required for each
 Column names and types for table `Samples`.
 
 <img
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)